packages/react/src/SelectPanel/SelectPanel.docs.json

```diff
@@ -1,5 +1,6 @@
 {
   "id": "select_panel",
+  "docsId": "select_panel",
```

Comment:
@broccolinisoup Do we need docsId on both v1 and v2 docs.json?

Reply:
Yes please. We need the unique docsId in all components that we want to group together.